### PR TITLE
chore: Clean up 2.0 comments

### PR DESCRIPTION
**Pull request by marc2332**
Depends on https://github.com/iotaledger/firefly/pull/7909
Closes https://github.com/iotaledger/firefly/issues/7949

### CODE REVIEW

File: packages/shared/lib/core/api/interfaces/api.interface.ts [hunk 1]
```diff
@@ -13,7 +13,6 @@ import {
 } from '@iota/sdk/out/types'
 import { IWallet } from '@core/profile/interfaces'
 
-// TODO(2.0): Every method should return a promise (maybe except Utils, needs research)
 export interface IApi {
```
Comment (cpl121) [inline packages/shared/lib/core/api/interfaces/api.interface.ts:16]: It's weird that `computeFoundryId` is a promise and `computeAccountId` and `computeNftId` are not. Is this correct?
On the other hand, doesn't `clearWalletsFromMemory` need a promise?

File: packages/shared/lib/core/profile/stores/active-wallets.store.ts [hunk 2]
```diff
@@ -23,7 +27,7 @@ export const nonHiddenActiveWallets: Readable<IWalletState[]> = derived([activeW
         return []
     }
     const unsortedNonHiddenwallets = $activeWallets?.filter((wallet) => !wallet?.hidden)
-    return unsortedNonHiddenwallets // TODO(2.0): Sort them: .sort((a, b) => a.index - b.index)
+    return unsortedNonHiddenwallets
```
Comment (cpl121) [inline packages/shared/lib/core/profile/stores/active-wallets.store.ts:30]:
```suggestion
    return $activeWallets?.filter((wallet) => !wallet?.hidden)
```

File: packages/shared/lib/core/profile/stores/active-wallets.store.ts [hunk 3]
```diff
@@ -36,6 +40,6 @@ export const visibleActiveWallets: Readable<IWalletState[]> = derived(
             $activeProfile?.showHiddenWallets ?? false
                 ? $activeWallets
                 : $activeWallets?.filter((wallet) => !wallet?.hidden)
-        return unsortedVisiblewallets // TODO(2.0): Sort them: .sort((a, b) => a.index - b.index)
+        return unsortedVisiblewallets
```
Comment (cpl121) [inline packages/shared/lib/core/profile/stores/active-wallets.store.ts:43]: `unsortedVisiblewallets` -> `visibleWallets` wdyt?

**Review by cpl121 — APPROVED** [review]
Cool

**Review by evavirseda — APPROVED** [review]
very clean :broom:

**Review by cpl121 — APPROVED** [review]

**Pull request merged**